lib/msal-core/karma.conf.js

```diff
@@ -9,12 +10,14 @@ module.exports = function (config) {
             'tests/**/*.ts',
         ],
         exclude: [
+            'tests/endToEndTests/**'
```

Comment:
I understand the intent of this exclusion. but instead of this, shouldn't we include what paths are required specifically?